wandb/sdk/data_types.py

```diff
@@ -674,8 +674,8 @@ class Object3D(BatchableMedia):
         data_or_path: (numpy array, string, io)
             Object3D can be initialized from a file or a numpy array.
 
-            The file types supported are obj, gltf, babylon, stl.  You can pass a path to
-                a file or an io object and a file_type which must be one of `'obj', 'gltf', 'babylon', 'stl'`.
+            You can pass a path to a file or an io object and a file_type
+            which must be one of `"obj", "gltf", "glb", "babylon", "stl", "pts.json".
```

Comment:
Could you wrap each of the strings with a backtick, like `"so"`?

This issue predates your PR (see [current docs](https://docs.wandb.ai/ref/python/data-types/object3d)), but should fix it while we're here.